.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Oh, wait, this file is not executable (`+x`), could you fix that?

`git update-index --chmod=+x .husky/pre-commit` should do the trick.